frontend/app/src/components/dashboard/OverallBalances.vue

```diff
@@ -173,7 +177,7 @@ const indicator = computed(() => {
   if (get(isLoading)) {
     return '';
   }
-  return get(balanceDelta).isNegative() ? '▼' : '▲';
+  return get(balanceDelta).isNegative() ? 'mdi-menu-down' : 'mdi-menu-up';
```

Comment:
why change the unicode character, was there any problem with them?

Reply:
I just think it was too big